features/display_dishes.feature

```diff
@@ -0,0 +1,17 @@
+Feature: As a visitor,
+  In order to see example of dishes available in general,
+  I need to be able to see a selection of dishes
+
+Background:
+  Given the following dishes exists
+    |    name   |                        description                              |  price  | readytime | portions |
+    | meatballs |    homecooked with love, including mashed potatoes and sauce    |    49   |   18:00   |    10    |
+    |   taco    |    really spicy authentic Mexican tacos                         |    59   |   16:00   |    10    |
```

Comment:
Also, I can't see that you are testing for the presence of `tacos`. If not used, why is it added to the db`